spec/models/reports/facility_state_spec.rb

```diff
@@ -617,15 +617,23 @@
     end
   end
 
-  xcontext "monthly hypertension overdue patients", run_with_custom_date: true do
+  context "monthly hypertension overdue patients", run_with_custom_date: true do
     describe "overdue_patients" do
       it "should return number of overdue patients assigned to the facility at beginning of a month" do
         month_date = reporting_dates[:beginning_of_month]
         facility = create(:facility)
         _overdue_patients = create_list(:patient, 2, :with_overdue_appointments, assigned_facility: facility, registration_facility: facility)
         _patient = create(:patient, :with_appointments, appointment_creation_date: month_date - 1, scheduled_date: month_date + 15.days, assigned_facility: facility, registration_facility: facility)
 
-        RefreshReportingViews.refresh_v2
+        views = %w[
```

Comment:
Can we move views to a let block to avoid duplication?

Reply:
yeah, make sense